modVM/src/org/aion/vm/ExecutionHelper.java

```diff
@@ -203,4 +161,33 @@ public void merge(ExecutionHelper other, boolean success) {
             addLogs(other.getLogs());
         }
     }
+
+    /**
+     * Returns the list of deleted accounts, the accounts to be deleted following a transaction.
+     *
+     * @return the deleted accounts list.
+     */
+    public List<Address> getDeleteAccounts() { return new ArrayList<>(deleteAccounts); }
+
+    /**
+     * Returns the execution logs.
+     *
+     * @return the execution logs.
+     */
+    public List<Log> getLogs() { return logs; }
```

Comment:
Maybe overly-paranoid, but would it be worth returning Collections.unmodifiableList(logs) instead?  That wrapper prevents someone from getting the logs and then modifying it, affecting the behaviour in this object